thunder/torch/__init__.py

```diff
@@ -1801,6 +1801,17 @@ def gelu(a: TensorProxy, /, *, approximate: str = "none") -> TensorLike:
         raise ValueError(f"gelu does not support the approximate={approximate} argument")
 
 
+@torchsymbol(torch.nn.functional.leaky_relu, is_method=False)
+def leaky_relu(a: TensorProxy, /, negative_slope=0.01, inplace: bool = False) -> TensorLike:
```

Comment:
Would it make sense to include all parameters?
 
https://pytorch.org/docs/stable/generated/torch.ao.nn.quantized.functional.leaky_relu.html#torch.ao.nn.quantized.functional.leaky_relu

Reply:
The link is for `torch.ao.nn.quantized.functional.leaky_relu`. Docs search in PyTorch is not great with many similar results. Do you suggest supporting the `torch.ao.nn.quantized.functional` namespace? Or is it just the confusion with links?
Here's the link for the `torch.nn.functional.leaky_relu` function that is implemented here: https://pytorch.org/docs/stable/generated/torch.nn.functional.leaky_relu.html#torch.nn.functional.leaky_relu

Reply:
I was following https://pytorch.org/docs/stable/generated/torch.nn.functional.leaky_relu.html#torch.nn.functional.leaky_relu 

What are the plans to support torch.ao.nn.quantized.functional?

Reply:
Yeah, I'm stupid, sorry.  :)